test/functional/codecept.conf.js

```diff
@@ -1,10 +1,14 @@
 exports.config = {
   tests: './*_test.js',
   output: './output',
+  timeout: 10000,
   helpers: {
     Puppeteer: {
+      show: true,
       url: 'http://localhost:3000',
-      waitForNavigation: 'networkidle0',
+      waitForNavigation: ['load', 'domcontentloaded', 'networkidle0'],
+      waitForAction: 5000,
```

Comment:
I think we probably want to remove that as it makes the tests run quite slowly.

Reply:
agree, but this is functional test part - I removed it from e2e, which I was working on. I'll leave that config clean up to @poojithanag